docs/commands/ts.mrevrange.md

```diff
@@ -3,7 +3,7 @@
 Query a range across multiple time series by filters in reverse direction.
 
 ```sql
-TS.MREVRANGE fromTimestamp toTimestamp
+TS.MREVRANGE fromTimestamp toTimestamp          
```

Comment:
@LiorKogan Why you didn't add the [LATEST] here like you did in ts.get?
The same goes for TS.MRANGE

Reply:
I know. Later.